lightning/src/ln/channelmanager.rs

```diff
@@ -2484,7 +2492,7 @@ where
 		}
 	}
 
-	fn get_updated_funding_outpoint_commitment_signed_internal(&self, channel_lock: &mut ChannelLock<<SP::Target as SignerProvider>::Signer>, funding_outpoint: &OutPoint, channel_value_satoshis: u64, own_balance: u64) -> Result<CommitmentSigned, APIError> {
+	fn get_updated_funding_outpoint_commitment_signed_internal(&self, channel_lock: &mut ChannelLock<<SP::Target as SignerProvider>::Signer>, funding_outpoint: &OutPoint, channel_value_satoshis: u64, own_balance: u64) -> Result<(CommitmentSigned, u64), APIError> {
```

Comment:
I would document the return type now that we are returning a tuple.

It might be an even better idea to return a dedicated type so that we can name the `u64`, but I'm not sure if we are very keen on adding new types to this branch.

Reply:
Added a type and some comments, agreed it makes things more readable.